src/containers/Onboarding/CreateGroup.js

```diff
@@ -47,35 +47,50 @@ class CreateGroup extends Component {
       resolve(this.props.createGroup(fields));
     });
 
+    promise.then((group) => {
+      return new Promise((resolve) => {
```

Comment:
No need for a new promise, you can just do:

```javascript
promise.then((group) => {
  this.orderFrequencyFor(group, fields, 'delivery');
  return group;
})
```

Check this https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/Promise.jsm/Promise#then()